hudi-hadoop-common/src/test/java/org/apache/hudi/common/table/timeline/TestArchivedTimelineV1.java

```diff
@@ -746,4 +790,38 @@ private static org.apache.hudi.avro.model.HoodieCommitMetadata convertCommitMeta
     avroMetaData.getExtraMetadata().put(HoodieRollingStatMetadata.ROLLING_STAT_METADATA_KEY, "");
     return avroMetaData;
   }
+
+  @Test
+  void shouldReadArchivedFileFrom2025AndValidateContent() {
+    Path path = new Path(TestArchivedTimelineV1.class
+        .getResource("/archivecommits/.commits_.archive.681_1-0-1").getPath());
+
+    assertDoesNotThrow(() -> readAndValidateArchivedFile(path, metaClient));
+  }
+
+  @Test
+  void shouldReadArchivedFileFrom2022AndValidateContent() {
+    Path path = new Path(TestArchivedTimelineV1.class
+        .getResource("/archivecommits/.commits_.archive.1_1-0-1").getPath());
```

Comment:
Could we generate smaller artifacts of archival files?  I also see that they contain specific schema.  Let's make sure there is no sensitive information there.